src/EditorFeatures/Core/Implementation/ReassignedVariable/ReassignedVariableOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Options.Providers;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.UnderlineReassignment
+{
+    internal class ReassignedVariableOptions
+    {
+        public static PerLanguageOption2<bool> Underline =
+           new PerLanguageOption2<bool>(nameof(ReassignedVariableOptions), nameof(Underline), defaultValue: true,
```

Comment:
this will likely be set to `false` when this is submitted.  it will be an opt-in option.